.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,6 +28,7 @@ Make sure you have checked _all_ steps below.
 
 - [ ] In case of new functionality, my PR adds documentation that describes how to use it.
   - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
+  - All the functions and the classes in the PR contain docstrings that explain what it does
```

Comment:
I know python doesn't have public/private func. But sometimes we use underscore in the beginning of the func to indicate it is a public func. Not sure if it is a convention else where. But we could call `All the public functions ...`

Reply:
Sure I agree, will update that.